packages/@featherds/select/wdio/FeatherSelect.ts

```diff
@@ -0,0 +1,28 @@
+const OPTION = "[data-ref-id='feather-select-list'] li";
+export class FeatherSelect {
+  private selectSelector: string;
+  get select() {
+    return $(this.selectSelector);
+  }
+  constructor(_select: string) {
+    this.selectSelector = _select;
+  }
+  async selectIndex(index: number) {
```

Comment:
Ideally we would also like a function like selectByVisibleText (or similar).

`
await $(listSelector).waitForDisplayed();
 const result = await browser.execute(
       function (selector: any, value: any) {
         const e: any = document.querySelectorAll(selector);
         for (let i = 0; i < e.length; i++) {
           if (e[i].textContent === value) {
             e[i].click();
             return "Item selected";
           }
         }`